purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -515,16 +497,10 @@ class Purchases internal constructor(
         packageToPurchase: Package,
         listener: PurchaseCallback
     ) {
-        val purchaseOption = packageToPurchase.product.bestPurchaseOption
-        if (purchaseOption == null) {
-            // TODOBC5: Improve and move error message
-            errorLog("PurchasePackage: Product does not have any purchase option")
-            return
-        }
```

Comment:
I removed these checks since they were outdated. They were meant for INAPP products but we already are throwing an exception later